### ISSUE TITLE
Correct heading order for single audio result page

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1616]
## Problem

The heading order of single result pages (for example, https://search.openverse.engineering/audio/2ecd5631-c48c-4a5f-89c4-83c44dbbd365) is currently incorrect. `h2` and `h4` headings are skipped.

This issue was originally for both audio and image results, but then issue  #1698 was opened to track the image detail page.

## Description
The heading order should be logical (h1 - h6) and not skip levels. The styles should still match the design mockups.

**Comment by RARNE12** [comment]
@obulat , @rarne12 and @kickeyhickey will start working on this issue. You can assign this task to me. Have a good day.

**Comment by RARNE12** [comment]
@obulat I am unable to find the correct file this heading is located at. Can you please tell me what parent component has the file with this header tag (Single Results) tag is located? Please advise, I appreciate any assistance you can offer. Thank you in advance. 
cc: @kickeyhickey

**Comment by obulat** [comment]
@RARNE12, I think you couldn't find the file because the `audio` detail page is a `page`, not a `component`. So, you would need to look at the `pages` directory: `src/pages/audio/_id.vue`. 

This file doesn't have any headings, though, so you will need to look at the child components there: `src/components/VMediaInfo/VMediaReuse.vue`, `src/components/VAudioDetails/VAudioDetails.vue` and `src/components/VAudioDetails/VRelatedAudio.vue`.

**Comment by RARNE12** [comment]
@obulat, thank you for your quick response and suggestions. @kickeyhickey and I will start working on it at our earliest. Have a great day.
